src/components/SRNEntry.vue

```diff
@@ -233,32 +235,21 @@ export default {
         }
       }
     },
-    /* Before an element is removed from the userIDList array, the login message and input message are reset to default values. 
-       removeInputField() is called to remove the element. 
-       In most cases, this would be enough. But there is one edge case:
-      Let's say the user typed in ID 'A' and decides to add another entry. This means 'A' is authenticated and it is valid/invalid. 
-      The user types in the ID 'B' as the next entry and it is invalid. 
-      The user changed their mind and decided to delete this entry ('B'). At this point, the variables will be in the following state:
-        validateCount = 1
-        isCurrentUserValid = false
-      The state of the variables contradict the fact that 'A' is already authenticated. This means any of the two cases:
-      - A is valid, so validateCount = 0 and isCurrentUserValid = true
-      - A is invalid, so validateCount = 2 and isCurrentUserValid = false
-      So when the user clicks submit, 'A' is authenticated again, which is a behaviour we do not want. So to bypass this, validateCount is set to 2. 
-      (To better understand how this will help, understanding processForm() will help.) 
-      isCurrentUserValid flag is ignored because the value is already set in each user's valid flag. 
-      @params {Number} - index - index of input field to be removed */
+    /** This method is called whenever - button is clicked, to remove an input field
+     * @param {Number} index - index of input field to be removed
+     */
     removeField(index) {
       this.resetInvalidInputMessage;
```

Comment:
Again, if these are methods then why are they not invoked?
Or is it a feature of vue that for methods with no arguments, we don't have to write `()`?

Reply:
Yeah, you don't need the brackets if you don't have any arguments. Actually, it doesn't matter even if you have arguments, it sends the whole event object gets sent, so arguments included. Only when you have default values to those arguments, then the () make a difference.

Reply:
Need to fix this and all the other places where you're trying to invoke a method but the method is not invoked because of missing brackets